resources/TrackingTabComponents/TrackingSkyPlotTab.qml

```diff
@@ -33,11 +33,61 @@ Item {
 
     property alias all_series: trackingSkyPlotPoints.all_series
     property var series: []
+    property var labelObjs: []
     property bool labelsVisible: labelsVisibleCheckBox.checked
     property bool polarChartWidthChanging: false
 
+    function clearLabels() {
```

Comment:
@pcrumley yeah it actually introduced a bug (no longer getting as frequent repainting of labels) so the timer sleep was making the labels flash.

this should be fixed now, also less refreshing now